### ISSUE TITLE
bottom-aligned tooltip in practice is covered by the mouse pointer

### ISSUE THREAD

**Issue by gashcrumb** [redhat-ipaas/ipaas-ui#506]
Just a note on the page design for the integration list and dashboard, the bottom-aligned tooltip doesn't work in practice as it's covered by the mouse pointer.  It'd be better to have the tooltip top-aligned.

Unfortunately I can't get a good screenshot of this in action though :-)

**Comment by kahboom** [comment]
I agree that top placement is better, but I moved it to the bottom because one error was so huge it literally overflowed into the browser URL bar (well was like `overflow: hidden` so you couldn't even read the error), if that makes sense. We would have to concatenate the message or tell the API devs to make the message relatively short.

**Comment by gashcrumb** [comment]
Agreed, betcha that case you ran into was a java stack trace :-)
